target/linux/ramips/dts/mt7621_winstars_ws-wn583a6.dts

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "winstars,ws-wn583a6", "mediatek,mt7621-soc";
+	model = "Winstars WS-WN583A6";
+
+	aliases {
+		led-boot = &led_status_red;
+		led-failsafe = &led_status_red;
+		led-running = &led_status_blue;
+		led-upgrade = &led_status_red;
+		label-mac-device = &wifi2g;
```

Comment:
This doesn't work without mtd-mac-address. You will have to use label_mac in lower section of 02_network.

Reply:
Sorry, I didn't know how to test it. It should be fixed now.
```
root@OpenWrt:~# . /lib/functions/system.sh
root@OpenWrt:~# get_mac_label
XX:XX:XX:XX:XX:91
```